CHANGELOG.md

```diff
@@ -1,16 +1,32 @@
 # Confluent's Python client for Apache Kafka
 
-## v1.8.3
+## v1.9.0
 
-v1.8.3 is a maintenance release with the following fixes:
+This is a feature release:
+
+ - OAUTHBEARER OIDC support
+
+### Fixes
 
  - The warnings for `use.deprecated.format` (introduced in v1.8.2)
    had its logic reversed, which result in warning logs to be emitted when
    the property was correctly configured, and the log message itself also
    contained text that had it backwards.
    The warning is now only emitted when `use.deprecated.format` is set
    to the old legacy encoding (`True`). #1265
-
+ - Use `str(Schema)` rather than `Schema.to_json` to prevent fastavro
+   from raising exception `TypeError: unhashable type: 'mappingproxy'`. (Federico Fissore, #1156, #1197)
+ - Fix the argument order in the constructor signature for
+   AvroDeserializer/Serializer: the argument order in the constructor signature for AvroDeserializer/Serializer was altered in v1.6.1, but the example is not changed yet. (Duc Le Tu, #1263)
+ - We use primitive types retrieved from the Confluent Registry
+   and encountered an issue where _schema_loads would cause json deserialization errors by double quoting valid primitive declarations. Previous tests included incorrectly specified primitive declarations, according to the Avro spec primitive declarations are valid JSON documents, but they had been specified as strings of their type name with no quoting. The tests are fixed as well as the issue in _schema_loads.
+   It also fixed an issue with Avro union types. _schema_loads was incorrectly causing json serialization errors for unions because it included them on accident with its special-casing of primitive declarations. A check was added for json arrays to exclude them from the special casing. A check was also added later to ensure the _schema_name property was special-cased to None for unions. This should have no impact on names in the registry because _schema_name isn't used at all for the recommended subject name strategy with unions. (Dylan Richardson, #989)
+ - typo in produce API documentation. #1246
+ - log_cb in SerializingProducer and DeserializingConsumer docstring should be logger. #1135
```

Comment:
This should be rewritten to indicate what was done, not what needs to be done, but this is also a small doc fix so no need to include in cl.